index.js

```diff
@@ -106,7 +106,12 @@ var registerShortcut = function (keybinding, initialization) {
 
   try {
     var ret = globalShortcut.register(keybinding, function () {
-      mb.window.isVisible() ? mb.hideWindow() : mb.showWindow()
+      if (mb.window.isVisible()) {
+        return mb.hideWindow()
+      }
+      
+      mb.showWindow()
+      mb.window.focus()
```

Comment:
no, only when the window show it is setting the focus, at least in linux it is behaving properly it opens and put the focus on it, without this command it was not getting the focus